python/cuml/dask/ensemble/base.py

```diff
@@ -288,13 +316,59 @@ def get_combined_model(self):
 
         return internal_model
 
+    def apply_reduction(self, reduce, partial_infs, datatype, delayed):
+        def back_to_dask(array, datatype):
+            res = array.compute()

```

Comment:
Does this need to force a compute here? It seems like delayed is passed in the function we want everything to be lazy.. there is dask.array.from_delayed, right? I'm just wondering if users might be confused if this becomes eager computation when they passed in delayed=True

Reply:
Yes, indeed no calculations should happen before `compute` with `delayed=True`. I could finally make `dask.array.from_delayed` work with unknown shape size. Thanks for the tip.